obspy/clients/arclink/client.py

```diff
@@ -1114,6 +1114,10 @@ def get_inventory(self, network, station='*', location='*', channel='*',
             # To not use routing if
             #  (1) not network id is given,
             #  (2) virtual network (Station group) is requested
+            if route:
+                msg = ("Routing was requested but parameter 'netork' is '{}' "
```

Comment:
There is a small typo! 
Is this something which can be fixed directly in master?

Reply:
Yep. Just fix it in master directly. 👍

Reply:
Nevermind! You already corrected this in a later commit.